src/Message/CancelSubscriptionsRequest.php

```diff
@@ -175,6 +180,27 @@ public function setSubscriptionReferences($value)
         return $this->setParameter('subscriptionReferences', $value);
     }
 
+    /**
+     * Gets the reason the subscription was canceled
+     *
+     * @return null|string
+     */
+    public function getCancelReason()
+    {
+        return $this->getParameter('cancelReason');
+    }
+
+    /**
+     * Set the reason the subscription was canceled
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setCancelReason($value)
```

Comment:
Same as above re: type hints

Reply:
This repo needs to support PHP 5.3 which doesn't have typehints unfortunately (except for array)